CHANGELOG.md

```diff
@@ -13,6 +14,28 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent release-candidate(s), only the delta to the previous release-candidate
 * for a public release, the consolidated changes since the previous public release
 
+# r1.2
+
+## Release Notes
+
+This release contains the definition and documentation of
+* number-recycling v0.1.0
+
+The API definition(s) are based on
+* Commonalities v0.5.0
+* Identity and Consent Management v0.3.0
+
+## number-recycling v0.1.0
+
+**number-recycling v0.1.0 is the first public release version of the Number-Recycling API.**
+
+- number-recycling v0.1.0 API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/NumberRecycling/r1.2/code/API_definitions/number-recycling.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/NumberRecycling/r1.2/code/API_definitions/number-recycling.yaml&nocors)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/NumberRecycling/blob/r1.2/code/API_definitions/number-recycling.yaml)
+
+**Full Changelog**: https://github.com/camaraproject/NumberRecycling/compare/r1.1...r1.2
```

Comment:
The Full Changelog for the first public release is more than the changes since the release candidate, but from he very beginning of the repository ("for a public release, the consolidated changes since the previous public release")
```suggestion
**Full Changelog**: https://github.com/camaraproject/NumberRecycling/commits/r1.2
```

Hint: you can see the correct line if you try to "Generate release notes" within the draft release.